examples/fluid/dam_break_2d.jl

```diff
@@ -35,7 +35,8 @@ tank_size = (floor(5.366 * H / boundary_particle_spacing) * boundary_particle_sp
 fluid_density = 1000.0
 sound_speed = 20 * sqrt(gravity * H)
 state_equation = StateEquationCole(; sound_speed, reference_density=fluid_density,
-                                   exponent=1, clip_negative_pressure=false)
+                                   exponent=1, clip_negative_pressure=false,
+                                   background_pressure=0.0)
```

Comment:
This is a free surface simulation. Why would we use a background pressure there? That would just cause the fluid to expand or contract at the beginning of the simulation.

Reply:
I think I forgot to remove it again.